osmdroid/src/main/java/org/odk/collect/osmdroid/OsmDroidMapFragment.java

```diff
@@ -86,17 +87,25 @@ public class OsmDroidMapFragment extends Fragment implements MapFragment,
         LocationListener, LocationClient.LocationClientListener {
 
     // Bundle keys understood by applyConfig().
-    static final String KEY_WEB_MAP_SERVICE = "WEB_MAP_SERVICE";
-
-    @Inject
-    MapProvider mapProvider;
+    public static final String KEY_WEB_MAP_SERVICE = "WEB_MAP_SERVICE";
 
     @Inject
     ReferenceLayerRepository referenceLayerRepository;
 
     @Inject
     LocationClient locationClient;
 
+    @Inject
+    MapConfigurator mapConfigurator;
```

Comment:
This could use the constructor with `FragmentFactory` but I'll leave that change right now. I can do that later as part of #4943 - I already have a branch running for that that depends on this PR.